openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -32,6 +38,111 @@ def get_identity(self, data):
             return None
 
 
+class UserProfileReadOnlySerializer(serializers.Serializer):
```

Comment:
I can live with UserReadOnlySerializer, as long as the documentation is clear. I really dislike having Profile in the name because Profile is used elsewhere for 2 very different things (UserProfile model, and the Profile page, both of which are just part of what this serializer deals with).